guide/14-deep-learning/edge-detection.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>In our model, I have supported all vgg and resnet backbone so please mention in the guide as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>